completions.go

```diff
@@ -117,15 +117,31 @@ type CompletionOptions struct {
 	HiddenDefaultCmd bool
 }
 
+// CompletionChoice is a string that can be used for completions
+//
+// two formats are supported:
+//   - the name of the flag
+//   - the name of the flag with its description (separated by a tab)
```

Comment:
I don’t understand the two previous lines. 
Shell completions are not only for flag names but for arguments and flag values as well. 

How about just referring to “the completion choice” and “the completion choice with a textual description”